saleor/core/taxes.py

```diff
@@ -1,10 +1,18 @@
 from dataclasses import dataclass
 from decimal import Decimal
-from typing import List, Union
+from typing import TYPE_CHECKING, List, Optional, Union
 
 from django.contrib.sites.models import Site
+from django.core.cache import cache
 from prices import Money, MoneyRange, TaxedMoney, TaxedMoneyRange
 
+from saleor.app.models import App
+from saleor.webhook.event_types import WebhookEventType
```

Comment:
@mateuszgrzyb I think this one is now deprecated on `main` as it now distinguishes between sync and async webhooks!